src/Altinn.Platform/Altinn.Platform.Storage/Storage/Program.cs

```diff
@@ -42,6 +42,10 @@ public static void Main(string[] args)
             Host.CreateDefaultBuilder(args)
             .ConfigureWebHostDefaults(webBuilder =>
             {
+                webBuilder.UseKestrel();
```

Comment:
This should probably be commented out before we merge with master..

Reply:
fixed